ci/ci.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    std::process::exit(0);
+}
```

Comment:
This code depends on `std` and redundant! Much better to remove `std::process::exit(0)` at all and add `no_std`:

```suggestion
#![no_std]
fn main() {}
```